JavaPrograms/src/com/home/org/inheritance/challenge/Vehicle.java

```diff
@@ -0,0 +1,48 @@
+package com.home.org.inheritance.challenge;
+
```

Comment:
Write a java doc for this class for example This class Vehicle defines a vehicle with some properties like speed, enginepower etc.